libs/model/src/aggregates/token/DistributeSkale.command.ts

```diff
@@ -0,0 +1,73 @@
+import { type Command, ServerError } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import {
+  createPublicClient,
+  createWalletClient,
+  http,
+  TransactionExecutionError,
+} from 'viem';
```

Comment:
Can you put any Viem logic in util functions in `libs/evm-protocols`. Trying to keep Web3 dependencies out of models. Technically we have utils for creating the private wallet but it is currently Web3.js based so I'm fine if you want to just quickly wrap this however way you want in evm-protocols and I will take care of it later.